velox/exec/MergeJoin.cpp

```diff
@@ -1368,12 +1368,6 @@ RowVectorPtr MergeJoin::applyFilter(const RowVectorPtr& output) {
             decodedFilterResult_.valueAt<bool>(i);
 
         joinTracker_->processFilterResult(i, passed, onMiss, onMatch);
-
-        if (isAntiJoin(joinType_)) {
```

Comment:
What's the fix? It seems that we move the add rows into miss callback?

Reply:
@xiaoxmeng Yes, placing the code to add rows within the onMiss function ensures that a row is added after all matched rows have been processed.